extension/beats/doris/config.go

```diff
@@ -29,16 +29,17 @@ import (
 )
 
 type config struct {
-	Hosts               []string `config:"fenodes"`
-	HttpHosts           []string `config:"http_hosts"`
-	User                string   `config:"user" validate:"required"`
-	Password            string   `config:"password"`
-	Database            string   `config:"database" validate:"required"`
-	Table               string   `config:"table" validate:"required"`
-	LabelPrefix         string   `config:"label_prefix"`
-	LineDelimiter       string   `config:"line_delimiter"`
-	LogRequest          bool     `config:"log_request"`
-	LogProgressInterval int      `config:"log_progress_interval"`
+	Hosts               []string         `config:"fenodes"`
+	HttpHosts           []string         `config:"http_hosts"`
+	User                string           `config:"user" validate:"required"`
+	Password            string           `config:"password"`
+	Database            string           `config:"database" validate:"required"`
+	Table               string           `config:"table"`
+	Tables              []map[string]any `config:"tables"`
```

Comment:
In elasticsearch output plugin, `tables` config is used to specify multiple tables to select by filters. In our case, we can simply use two config: `table` and `default_table`.